src/libraries/System.Formats.Tar/tests/TarTestsBase.cs

```diff
@@ -611,5 +612,37 @@ public static IEnumerable<object[]> GetPaxAndGnuTestCaseNames()
                 yield return new object[] { name };
             }
         }
+
+        public static bool CanExtractWithTarTool = CanExtractWithTarTool_Method();
```

Comment:
nit, why isn't this simply a property instead of a field + method?

Reply:
This way it can be cached. It is not very important now because I am using it only in one test but if more tests start to use `CanExtractWithTarTool` and it was a property, each test would create a process, which is less desirable that doing it just once.

Reply:
Oh right. Yeah, often we just use Lazy or a null check. Whatever works